src/test/java/domain/LadderResultTest.java

```diff
@@ -0,0 +1,48 @@
+package domain;
+
+import java.util.List;
+import java.util.Map;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class LadderResultTest {
+
+    private LadderResult ladderResult;
+
+    @BeforeEach
+    void setLadderResult() {
+        ladderResult = new LadderResult(Map.of(
+                "사람1",
+                "보상1",
+                "사람2",
+                "보상2"
```

Comment:
```suggestion
                "사람1", "보상1",
                "사람2", "보상2"
```
이렇게 사용하면 key와 value가 더 명확히 보일것 같네요

Reply:
beforeEach를 사용해도 무방하지만 
테스트 메서드 별로 케이스를 적절히 만들어서 보여주는것도 더 가독성 좋은 테스트를 작성하는 방법중 하나라고 생각합니다!